projects/map/src/lib/openlayers-map/openlayers-map.ts

```diff
@@ -31,6 +31,7 @@ export class OpenLayersMap implements MapViewerModel {
 
   private readonly resizeObserver: ResizeObserver;
   private initialExtent: OpenlayersExtent = [];
+  private initialCenterZoom?: [number[], number] = undefined;
```

Comment:
I don't think this is needed. This is used when app config changes but then you probably don't want to zoom to the same location again.

Reply:
The issue is that the bookmark may be applied before the map loads and checks the initial extent, which I believe may be able to happen.